myscripts.js

```diff
@@ -0,0 +1,18 @@
+const burgerButton = document.querySelector('.fa-bars');
+const closeButton = document.querySelector('.mobile-menu-i');
+const mobileMenu = document.querySelector('.mobile-menu');
+// const body = document.querySelector('body');
```

Comment:
- [ ] Please remove the commented code before pushing the changes into the repo to making you code look unprofessional.